src/components/App.tsx

```diff
@@ -1849,11 +1952,41 @@ export class App extends React.Component<any, AppState> {
     }
   }
 
+  private savePointerDebounced = throttle(
```

Comment:
Can you rename to "Throttled" instead of "Debounced"?

Reply:
per discussion, I'm removing the throttle altogether.